test/runtests.jl

```diff
@@ -2058,4 +2058,92 @@ using Aqua: Aqua
             Aqua.test_project_toml_formatting(BSeries)
         end
     end
+
+    @testset "Continuous Stage Runge-Kutta Method" begin
+        @testset "(Inverse) 4x4-Hilbert Matrix" begin
+            # References
+            # - Yuto Miyatake and John C. Butcher.
+            # "A characterization of energy-preserving methods and the construction of
+            # parallel integrators for Hamiltonian systems."
+            # SIAM Journal on Numerical Analysis 54, no. 3 (2016): 
+            # [DOI: 10.1137/15M1020861](https://doi.org/10.1137/15M1020861) 
+
+            # This is the matrix obtained by inverting the 4x4-Hilbert matrix
+            M = [-6//5   72//5  -36//1  24//1;
+            72//5  -144//5 -48//1  72//1;
+            -36//1 -48//1   720//1 -720//1;
+            24//1   72//1  -720//1  720//1]
+            csrk = ContinuousStageRungeKuttaMethod(M)
+            # Generate the bseries up to order 4
+            order = 4
+            series = bseries(csrk, order)
+            expected_coefficients = [1//1 ,1//1, 1//2, 1//6, 1//3, 1//24, 1//12, 1//8, 1//4]
+            @test collect(values(series)) == expected_coefficients
+        end
+
+        @testset "Floating-points coefficients" begin
+            # Define variables
+            a = -0.37069987
+            b = 0.74139974
+            c = 2.51720052
+
+            # Create symbolic matrix
+            Minv = [a 1//2 b 1//6;
+            1//2 1//4 1//6 1//8;
+            b 1//6 c 1//10;
+            1//6 1//8 1//10 1//12]
```

Comment:
Please check your comments everywhere. This is not a symbolic matrix since you set the coefficients above to fixed floating point values. Where did you get these special choices from? Please add some references

Reply:
Okey, I'll fix the comments. As for the values, I took them from the project I am working on with @ketch,  I calculated them